pkg/types/gcp/platform.go

```diff
@@ -35,11 +35,7 @@ type Platform struct {
 	// +optional
 	ComputeSubnet string `json:"computeSubnet,omitempty"`
 
-	// Licenses is a list of licenses to apply to the compute images
-	// The value should a list of strings (https URLs only) representing the license keys.
-	// When set, this will cause the installer to copy the image into user's project.
-	// This option is incompatible with any mechanism that makes use of pre-built images
-	// such as the current env OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE
+	// Licenses is no longer used and should be either empty or contain the nested virt license.
 	// +optional
 	Licenses []string `json:"licenses,omitempty"`
```

Comment:
Let's go ahead and remove `Licenses`. It has been deprecated long enough that we are safe to remove.